images/10-hypervvmtools/Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM ubuntu:16.04 as build-essential
+# FROM arm64=skip arm=skip
+RUN apt-get update  && \
+    apt-get install -y --no-install-recommends kmod bash git gcc make build-essential && \
+    rm -rf /var/lib/apt/lists/*
+
+ARG OS_KERNEL_VERSION
+ENV KERNEL_VERSION=${OS_KERNEL_VERSION}
+
+COPY headers.sh  /
+COPY build.tar.gz /
+
+RUN /headers.sh && \
+    git clone git://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable.git && \
+    cd linux-stable/tools/hv && \
+    make && \
+    apt-get remove -y --allow-remove-essential kmod bash git gcc make build-essential && \
+    rm -rf /build.tar.gz && \
+    cp hv_kvp_daemon /usr/bin && \
+    cp hv_vss_daemon /usr/bin && \
+    cp hv_fcopy_daemon /usr/bin && \
+    mkdir -p /usr/libexec/hypervkvpd/ && \
+    cp hv_get_dhcp_info.sh /usr/libexec/hypervkvpd/hv_get_dhcp_info && \
+    cp hv_get_dns_info.sh /usr/libexec/hypervkvpd/hv_get_dns_info && \
+    cp hv_set_ifconfig.sh /usr/libexec/hypervkvpd/hv_set_ifconfig && \
+    chmod +x /usr/libexec/hypervkvpd/* && \
+    cp lsvmbus /usr/bin && \
+    cd / && \
+    rm -rf linux-stable
+
+FROM ubuntu:16.04
+# FROM arm64=skip arm=skip
+
+RUN apt-get update  && \
+    apt-get install -y --no-install-recommends sudo && \
+    rm -rf /var/lib/apt/lists/* && \
+    mkdir -p /usr/libexec/hypervkvpd/
+
+COPY --from=build-essential /usr/bin/hv_* /usr/bin/
+COPY --from=build-essential /usr/bin/lsvmbus /usr/bin/
+COPY --from=build-essential /usr/libexec/hypervkvpd/* /usr/libexec/hypervkvpd/
+
+RUN chmod +x /usr/bin/hv_* && \
+    chmod +x /usr/bin/lsvmbus
+
+ENTRYPOINT ["hv_kvp_daemon", "-n"]
```

Comment:
`ENTRYPOINT ["/usr/bin/ros", "entrypoint"]` should be better.

Reply:
OK